kite-data/kite-data-core/src/main/java/org/kitesdk/data/Datasets.java

```diff
@@ -31,25 +31,52 @@
 import org.kitesdk.data.spi.Registration;
 
 /**
- * <p>Convenience methods for working with {@link Dataset} instances.</p>
- *
+ * Methods for working with {@link Dataset} instances.
+ * <p>
+ * <b>URIs</b><br />
+ * All methods require a URI that identifies a dataset, view, or
+ * repository. The URI must begin with the scheme {@code dataset:}, 
+ * {@code view:}, or {@code repo:}. The remainder of the URI is 
+ * implementation specific, depending on the dataset scheme.
+ * <p>
+ * For example, the URI {@code dataset:hive:/movies/ratings} 
+ * references a Hive dataset named <i>ratings</i> in the
+ * <i>movies</i> namespace.
+ * <p>
+ * The URI {@code view:hdfs:/user/myfiles/movies/ratings?year=2015&month=3}
```

Comment:
This uses "myfiles" where you would normally see a user name, implying that the user is actually named "myfiles". I think it would be better to use a generic user name there. I've been using "/user/me/movies/ratings".